packages/amplify-cli/src/domain/amplify-usageData/UsageDataPayload.ts

```diff
@@ -19,6 +19,7 @@ export class UsageDataPayload implements IUsageDataPayload {
   inputOptions: InputOptions;
   timestamp: string;
   error!: SerializableError;
+  downstreamException!: SerializableError;
```

Comment:
will there always be downstream exception when we initialize ```UsageDataPayload``` ?

Reply:
No, it's not a non null assertion operator here but a definitive assignment assertion operator since it's not being initialized in the constructor(see the `OkGreeter` class [here](https://www.typescriptlang.org/docs/handbook/2/classes.html#fields))

It can be null, and there is a unit test for that as well (works exactly similar to `Error` which can also be null)